symbolic-base/src/ZkFold/Base/Protocol/Protostar/Fold.hs

```diff
@@ -173,6 +152,7 @@ iterate
     => Scale (FieldElement ctx) comm
     => ctx ~ ArithmeticCircuit a (Vector n)
     => SPS.Input (FieldElement ctx) (C n a) ~ Vector n (FieldElement ctx)
+    => SymbolicInput (Vector n (FieldElement (ArithmeticCircuit a ((Vector n :.: Par1) :*: U1))))
```

Comment:
I'd just add the `SymbolicInput x => SymbolicInput (Vector n x)` instance in `ZkFold.Symbolic.Data.Input` module